README.md

```diff
@@ -207,7 +208,9 @@ Those files are located in the /built-flexy-java-app/ directory of the release z
 ### Configuration
 At startup of the Flexy Java application component, it checks for a configuration file in the /usr/ directory of the Ewon. If one is not present, the application will automatically create one with default values.
 
-After the Flexy Java application component has been installed and started, the configuration should be modified to contain the proper values for AppKey and ThingworxIP for your Thingworx instance.
+After the Flexy Java application component has been installed and started, the configuration should be modified to contain the proper values for AppKey and Thingworx Base URL for your Thingworx instance.
+### Thingworx Base URL 
+This parameter in the config file should be changed to the Base URL for the target Thingworx REST API. Users should pay close attention to the scheme designated in the URL. Specifying the scheme "https" will ensure all traffic to the server is encrypted over TLS. For example, setting the ThingworxBaseUrl parameter to "https:example.com:8443" will cause the connector to host "example.com" on port 8443 using TLS/SSL. It's possible to only specify the IP address of the Thingworx instance. This configuration will assume "http" scheme and communication will not be encrypted. 
```

Comment:
should it be " to "https:example.com:8443" will cause the connector **to connect** to host "example.com" on port 8443 using TLS/SSL"

Reply:
I went with " will create a connection to host", didn't like "connector to connect"

Reply:
Yeah, that phrasing is much better.